gravitee-apim-portal-webui-next/src/app/api-details/api-tab-subscriptions/subscriptions-details/subscriptions-details.component.html

```diff
@@ -0,0 +1,139 @@
+      <mat-card-header>
+        <div class="m3-title-medium subscriptions-details__header" i18n="@@subscriptionDetailsHeader">Subscription</div>
+      </mat-card-header>
+      <mat-card-content>
+        <div class="subscriptions-details__content-row">
+          <p
+            class="m3-title-small subscriptions-details__content-row-title subscriptions-details__content-row-app"
+            i18n="@@subscriptionDetailsHeaderApp">
+            App:
+          </p>
+          <p class="m3-body-medium subscriptions-details__content-row-app">{{ details.application }}</p>
+        </div>
+        <div class="subscriptions-details__content-row subscriptions-details__content-info">
+          <p class="m3-title-small subscriptions-details__content-row-title" i18n="@@subscriptionDetailsHeaderPlan">Plan:</p>
+          <p class="m3-body-medium">{{ details.plan }}</p>
+        </div>
+        <div class="subscriptions-details__content-row subscriptions-details__content-info">
+          <p class="m3-title-small subscriptions-details__content-row-title" i18n="@@subscriptionDetailsHeaderAuthentication">
+            Authentication:
+          </p>
+          <p class="m3-body-medium">{{ details.authentication }}</p>
+        </div>
+        @if (details.status) {
+          <div class="subscriptions-details__content-row">
+            <p class="m3-title-small subscriptions-details__content-row-title" i18n="@@subscriptionDetailsSubscription">Subscription:</p>
+            <p class="m3-body-medium">{{ details.status | capitalizeFirst }}</p>
+          </div>
+        }
+      </mat-card-content>
+    </mat-card>
+    <mat-card appearance="outlined" class="subscriptions-details__card">
+      @if (details.status === 'PENDING') {
+        <div class="m3-title-medium subscriptions-details__pending">
+          <div class="m3-body-large" i18n="@@subscriptionDetailsPendingHeader">Subscription in progress</div>
+          <div class="m3-body-small" i18n="@@subscriptionDetailsPendingContent">
+            Your subscription request is being validated. Come back later.
+          </div>
+        </div>
+      } @else if (details.status === 'REJECTED') {
+        <div class="m3-title-medium subscriptions-details__rejected">
+          <div class="m3-body-large" i18n="@@subscriptionDetailsRejectedHeader">Subscription rejected</div>
+        </div>
+      } @else if (details.status === 'PASUED') {
+        <div class="m3-title-medium subscriptions-details__rejected">
+          <div class="m3-body-large" i18n="@@subscriptionDetailsRejectedHeader">Subscription paused</div>
+        </div>
+      } @else if (details.status === 'CLOSED') {
+        <div class="m3-title-medium subscriptions-details__rejected">
+          <div class="m3-body-large" i18n="@@subscriptionDetailsRejectedHeader">Subscription closed</div>
+        </div>
+      } @else if (details.security === 'OAUTH2' || details.security === 'JWT') {
```

Comment:
So after this `if` it's for accepted status only ? this condition is not very easy to read. 😅

Reply:
Done